core/src/main/kotlin/plugability/DokkaContext.kt

```diff
@@ -124,7 +124,7 @@ private class DokkaContextConfigurationImpl(
     }
 
     private fun findNotOverridden(bucket: Set<Extension<*, *, *>>): Extension<*, *, *> {
-        val filtered = bucket.filter { it !in suppressedExtensions }
+        val filtered = bucket.filter { it !in suppressedExtensions.filter { it.value.all { it is Suppression.ByExtension } } }
```

Comment:
> Is bucket.filterNot(..) better?

filterNot with any and `!is` will be more efficient since it will abort on the first occurrence. 

> all or any?

It has to be filter + all or filetrNot + any + `!is` since we want only suppressedExtensions by overrides. If there is any suppression other than overrides we handle it in a separate method. It was breaking the previous implementation when we had only `Suppresion.ByPlugin` by manually disabling the extensions.